admin/lib/consts/service-roles.js

```diff
@@ -0,0 +1,6 @@
+module.exports = Object.freeze({
+  helpdesk: 'HELPDESK',
+  serviceManager: 'TEACHER',
+  teacher: 'TEACHER',
+  testDeveloper: 'TEST-DEVELOPER'
+})
```

Comment:
Whilst it is much better having the object 👍 I think the serviceManager role needs to be set to SERVICE-MANAGER 😉

Reply:
oh wow that was a good one. resolved!